src/table/table.scss

```diff
@@ -57,14 +57,14 @@ $hover-scrollbar-color: white !default;
 
   // selection/hover
   &.selectable {
-    .public_fixedDataTableCell_cellContent {
+    .public_fixedDataTableRow_main {
```

Comment:
If there is a row main class couldn't we use it below instead of row_odd and row_even?